contrib/ros/envs/example_launchers/trpo_gazebo_sawyer_pnp.py

```diff
@@ -8,13 +12,46 @@
 from sandbox.rocky.tf.policies.gaussian_mlp_policy import GaussianMLPPolicy
 from sandbox.rocky.tf.envs.base import TfEnv
 
+from contrib.ros.envs.example_launchers import model_dir
 from contrib.ros.envs.sawyer.pick_and_place_env import PickAndPlaceEnv
+from contrib.ros.util.task_object_manager import TaskObject, TaskObjectManager
 
 
 def run_task(*_):
+    block = TaskObject(
+        name='block',
+        initial_pos=Point(x=0.5725, y=0.1265, z=0.90),
```

Comment:
Sorry I missed that in the previous PR but I think it is a bit inconvenient to have the user import `geometry_msgs.msg`. Can you make the interface use arrays instead (just like you define `initial_goal` for example)?